docs/en/latest/apisix-ingress-controller.md

```diff
@@ -34,7 +34,7 @@ To install apisix-ingress-controller which release name is `apisix-ingress-contr
 ```bash
 helm repo add apisix https://charts.apiseven.com
 helm repo update
-helm install apisix-ingress-controller apisix/apisix-ingress-controller --namespace ingress-apisix
+helm install apisix-ingress-controller apisix/apisix-ingress-controller --namespace ingress-apisix --create-namespace
```

Comment:
ref https://github.com/apache/apisix-helm-chart/issues/176, we need add ` --set ingress-controller.config.apisix.serviceNamespace=<your namespace>` flag.

Reply:
is this another `topic`? the `--create-namespace` only fixes the `namespace` not exists issue, not others.

Reply:
Got it.